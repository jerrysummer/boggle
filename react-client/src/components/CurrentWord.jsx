import React from 'react';


class CurrentWord extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            items: []
        }
    }
    render() {
        return (
            <div>
                <h5>current Selection</h5>
                <h6>{this.props.currentSelection}</h6>
                <button onClick={this.props.handleWordSubmit}>
                    submit
                </button>
            </div>
        );
    }
}

export default CurrentWord;