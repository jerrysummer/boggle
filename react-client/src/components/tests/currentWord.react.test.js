import React from 'react';
import renderer from 'react-test-renderer';
import CurrentWord from '../CurrentWord';

it('renders correctly', () => {
  const component = renderer
    .create(<CurrentWord board="testword" />)
    .toJSON();
  expect(component).toMatchSnapshot();
});

