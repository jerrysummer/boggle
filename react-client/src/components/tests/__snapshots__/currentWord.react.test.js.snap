// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="currentWord"
>
  <h5
    className="word"
  >
    Current Word : 
  </h5>
  <h5
    className="letters"
  />
  <input
    className="button"
    id="submit"
    onClick={undefined}
    type="submit"
    value="Submit Word"
  />
</div>
`;
